### PR TITLE
Serialization for Candidates and Experiments Data Frames

### PR DESCRIPTION
**Pull request by jduerholt**
We are currently integrating BoFire in a third-party application, and I thought that it makes sense for this purpose to have also the serializable data models for the experiments and candidates pandas dataframes directly in bofire.

### CODE REVIEW

**Comment by bertiqwerty** [comment]
> We are currently integrating BoFire in a third-party application, and I thought that it makes sense for this purpose to have also the serializable data models for the experiments and candidates pandas dataframes directly in bofire.

Thanks Johannes. Let me make a proposal with some changes.

**Comment by jduerholt** [comment]
Fine for me ;) Any timeframe for this, as I am under a bit of time pressure with it and already start to integrate it elsewhere :D As usual :D

File: bofire/data_models/dataframes/dataframes.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            outputs = {
+                k: CandidateOutputValue(
+                    value=row[f"{k}_pred"],
+                    objective_value=row[f"{k}_des"],
+                    standard_deviation=row[f"{k}_std"],
+                )
+                for k in domain.outputs.get_keys()
+            }
+        else:
+            print(row)
```
Comment (jduerholt) [inline bofire/data_models/dataframes/dataframes.py:124]: Need to remove this.

**Comment by bertiqwerty** [comment]
> Fine for me ;) Any timeframe for this? As I am under a bit of time pressure with it and already start to integrate it elsewhere :D As usual :D

I pushed it. Have a look.

**Comment by jduerholt** [comment]
Thank you very much. Looks good to me. I will mergeit in ;) And work with this version. I would also like to make a release. Fine for you?

**Review by bertiqwerty — APPROVED** [review]

**Pull request merged**